src/core/textures/BaseTexture.js

```diff
@@ -405,15 +499,24 @@ BaseTexture.fromImage = function (imageUrl, crossorigin, scaleMode)
             image.crossOrigin = determineCrossOrigin(imageUrl);
         }
 
+        var imageType = getImageTypeOfUrl(imageUrl);
+
+        if (imageType) {
+            image.type = imageType;
+        }
+
         baseTexture = new BaseTexture(image, scaleMode);
         baseTexture.imageUrl = imageUrl;
 
-        image.src = imageUrl;
-
-        utils.BaseTextureCache[imageUrl] = baseTexture;
+        baseTexture.sourceScale = sourceScale;
```

Comment:
Sorry I misclicked, I was referring to the lines below this:

```js
baseTexture.imageType = image.type;
```

`imageType` was the property my comment was about.